README.md

```diff
@@ -43,7 +44,13 @@ already defined in the Jenkins global configuration, an ephermal resource is
 used: These resources only exist as long as any running build is referencing
 them.
 
-Examples:
+- **resource** - The resource name to lock as defined in Global settings. If inversePrecedence isn't also specified, the step can be called as lock('some-resource') without the named argument. Set to `null` if locking on a label in a declarative pipeline is desired.
+- **label** - Can be used to require locks on multiple resources concurrently. The build will wait until all resources tagged with the given label are available. Only this or resource can be used simultaneously.
+- **quantity** - (optional) Specifies the number of resources required within the selected label. If not set, all resources from the label will be required.
+- **inversePrecedence** - (optional) By default waiting builds are given the lock in the same order they requested to acquire it. If inversePrecedence is set to true, this will be done in reverse order instead, so that the newest build to request the lock will be granted it first.
+- **variable** - (optional) When locking a resource via a label you can use variable to set an environment variable with the name of the locked resource.
+
```

Comment:
This part was taken from the old wiki page. I was wondering why it wasn't copied to the readme.

Reply:
Mostly, because its redundant, because all those options have context-sensitive help texts...